doc/swagger.v1.yml

```diff
@@ -286,6 +628,134 @@ definitions:
         description: when the transaction was entered
     required: ['id', 'user_id', 'user_name', 'amount', 'note', 'created_at']
 
+  ArticleCategory:
+    type: object
+    properties:
+      id:
+        type: integer
+      name:
+        type: string
```

Comment:
do we want to add `description` to all properties?

Reply:
Good idea. Except, perhaps `id` and `name` (which are self-explanatory).